pyradiosky/skymodel.py

```diff
@@ -5411,7 +5438,9 @@ def read_votable_catalog(
     )
 
     if return_table:
-        return skyobj.to_recarray()
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            return skyobj.to_recarray()
```

Comment:
I see that this function as a whole is deprecated, So we're not too worried about issuing yet another deprecation inside of it I presume?

Reply:
exactly

Reply:
Oh, maybe I misunderstood your question. I just figured we didn't need multiple deprecation warnings, but I can remove this filter.

Reply:
Oh no I think you understood it correctly. There's a few like this so I just wanted to double check